pkg/image/admission/imagepolicy/api/v1/default-policy.yaml

```diff
@@ -0,0 +1,20 @@
+kind: ImagePolicyConfig
+apiVersion: v1
+executionRules:
+- name: execution-denied
+  # Reject all images that have the annotation images.openshift.io/deny-execution set to true.
+  # This annotation may be set by infrastructure that wishes to flag particular images as dangerous
+  onResources:
+  - resource: pods
+  - resource: builds
+  reject: true
+  matchImageAnnotations:
+  - key: images.openshift.io/deny-execution
+    value: "true"
+  allowResolutionFailure: true
```

Comment:
I.e. resolve changes the whole behavior of the platform.  I might have
enabled it last week - but right now it would require a retest of every
failure mode.

On Wed, Aug 10, 2016 at 10:03 AM, Clayton Coleman <ccoleman@redhat.com>
wrote:

> I don't think we should resolve without much deeper testing.  It's a more
> invasive change.
>
> On Wed, Aug 10, 2016 at 9:50 AM, David Eads <notifications@github.com>
> wrote:
>
>> In pkg/image/admission/imagepolicy/api/v1/default-policy.yaml
>> <https://github.com/openshift/origin/pull/8995#discussion_r74247302>:
>>
>> > @@ -0,0 +1,20 @@
>> > +kind: ImagePolicyConfig
>> > +apiVersion: v1
>> > +executionRules:
>> > +- name: execution-denied
>> > +  # Reject all images that have the annotation images.openshift.io/deny-execution set to true.
>> > +  # This annotation may be set by infrastructure that wishes to flag particular images as dangerous
>> > +  onResources:
>> > +  - resource: pods
>> > +  - resource: builds
>> > +  reject: true
>> > +  matchImageAnnotations:
>> > +  - key: images.openshift.io/deny-execution
>> > +    value: "true"
>> > +  allowResolutionFailure: true
>>
>> I'd like this to resolve: true too.
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/openshift/origin/pull/8995/files/5acd1663143e0c8196692974691537db1b40cc53#r74247302>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/ABG_p2wTLrmv4FCvfIDNAbk-nQe1UdFoks5qedc4gaJpZM4IlDlb>
>> .
>>
>
>